ckanext/twitter/plugin.py

```diff
@@ -38,17 +34,31 @@ def update_config(self, config):
         # Add resources
         p.toolkit.add_resource('theme/fanstatic', 'ckanext-twitter')
 
-    # IResourceController
+
+    # IResourceController, called after a resource is created
+    def after_create(self, context, res_pkg_dict):
+        data_dict = {'id': res_pkg_dict['package_id']}
```

Comment:
The variable `res_pkg_dict` should probably be `resource` to align with other CKAN code.